### DISCUSSION TITLE
Text Highlight Color

### DISCUSSION THREAD

**Discussion by tance77** [tailwindlabs/tailwindcss#4563] [Ideas]
I was wondering if we could get a feature to allow us to change the selected text highlight color.

```
::-moz-selection { /* Code for Firefox */
  color: red;
  background: yellow;
}

::selection {
  color: red;
  background: yellow;
}
```

Maybe something like `text-highlight-blue-500`

Going to throw good old w3 schools in here as an [example](https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_text_selection) because it was the first result on google.

**Comment by VicGUTT** [comment]
Hey ! Well, you're in luck as it is coming to Tailwind v2.2 (1 or 2 weeks from now). Check out [Adam's tweet on the matter](https://twitter.com/adamwathan/status/1397655318205714442).

I've put out a quick plugin for you to see how it could be achieved now : https://play.tailwindcss.com/uGnmPPcEhz?file=config

Hope it helps :v:
